### ISSUE TITLE
Migrate to travis-ci.com

### ISSUE THREAD

**Issue by choweiyuan** [cdancy/jenkins-rest#252]
### Expected Behavior
CI build should run

### Current Behavior
CI build seems to be failing on travis-ci.org since last year, 
https://blog.travis-ci.com/2021-05-07-orgshutdown
https://travis-ci.org/github/cdancy/jenkins-rest

Migrate procedure here: https://docs.travis-ci.com/user/migrate/open-source-repository-migration

**Comment by cdancy** [comment]
@choweiyuan I honestly have not been looking at the CI builds here and instead just run them locally when I need to do a release.

I know @martinda has played around with this before and might be able to spot something off.

**Comment by lemeurherve** [comment]
As an alternative, this project could be hosted and built on Jenkins infrastructure if you want.

**Comment by martinda** [comment]
I had a [fork](https://github.com/jrestclients/jenkins-rest) (now disabled and possibly hidden) that used the github actions to build it. I don't now how to get it built on Jenkins. Do you mean ci.jenkins.io ? That's only for the Jenkins community. But if this project is build on Jenkins, I would recommend building all the cdancy/*-rest project the same way.

**Comment by cdancy** [comment]
@lemeurherve yeah if there is an easy way to get this up and running on Jenkins somewhere that would be wonderful I just personally don't know how to do so.

**Comment by lemeurherve** [comment]
@cdancy The process isn't difficult: https://www.jenkins.io/doc/developer/publishing/requesting-hosting/

> Once you’ve completed all steps described on this page you will have accomplished the following:
> - Your plugin source code will be hosted in a repository in the jenkinsci GitHub organization. You will have admin access to this repository.
> - You will be allowed to release the plugin to the Jenkins project Maven repository, which serves as a source for the Jenkins project operated update sites.

I understand you might want to keep this repository here, but if you're interested nonetheless I can assist you.

**Comment by martinda** [comment]
@lemeurherve would this offer also stand for the other libraries such as [cdancy/artifactory-rest](https://github.com/cdancy/artifactory-rest) and [cdancy/bitbucket-rest](https://github.com/cdancy/bitbucket-rest) ?

**Comment by timja** [comment]
> @lemeurherve would this offer also stand for the other libraries such as [cdancy/artifactory-rest](https://github.com/cdancy/artifactory-rest) and [cdancy/bitbucket-rest](https://github.com/cdancy/bitbucket-rest) ? As a user and contributor, it would benefit me if they were all managed the same way.

Hi, no it would just be for the Jenkins one.
